app-backend/src/template/server-block-conf.ejs

```diff
@@ -58,6 +58,7 @@ if (!s.forceSsl || s.hasSsl) {
             proxy_set_header X-Real-IP $remote_addr;
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header X-Forwarded-Proto $scheme;
+            proxy_http_version 1.1;
```

Comment:
As far as I understand, this is activating gzip for internal communication between two services on the same machine. It has nothing to do with gzipping between the client and the server. If my assumption is correct, this is not a good change, gzipping comes with CPU usage. The bandwidth that is being saved due to gzipping is the local bandwidth (within the server) which is quite cheap. So we are consuming CPU time for zipping and unzipping data on the same machine.